_aggregations/pipeline/max-bucket.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: default
+title: Maximum bucket
+parent: Pipeline aggregations
+nav_order: 100
+redirect_from:
+  - /query-dsl/aggregations/pipeline-agg#avg_bucket-sum_bucket-min_bucket-max_bucket/
+  - /query-dsl/aggregations/pipeline/max-bucket/
+---
+
+# Maximum bucket aggregations
+
+The `max_bucket` aggregation is a sibling aggregation that calculates the maximum of a metric in each bucket of a previous aggregation.
+
+The specified metric must be numeric and the sibling aggregation must be a multi-bucket aggregation.
+
+## Parameters
+
+The `max_bucket` aggregation takes the following parameters.
+
+| Parameter             | Required/Optional | Data type       | Description |
+| :--                   | :--               |  :--            | :--         |
+| `buckets_path`        | Required          | String          | The path of the aggregation buckets to be aggregated. See [Pipeline aggregations]({{site.url}}{{site.baseurl}}/aggregations/pipeline/index#pipeline-aggregation-syntax). |
+| `gap_policy`          | Optional          | String          | The policy to apply to missing data. Valid values are `skip`, `insert_zeros`, and `keep_values`. Default is `skip`. |
+| `format`              | Optional          | String          | A [DecimalFormat](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/text/DecimalFormat.html) formatting string. Returns the formatted output in the aggregation's `value_as_string` property. |
+
+## Example
+
+The following example creates a date histogram with a one-month interval from the OpenSearch Dashboards e-commerce sample data. The `sum` sub-aggregation calculates the sum of all bytes for each month. Finally, the `max_bucket` aggregation finds the maximum---the largest of these buckets:
+
+```json
+POST opensearch_dashboards_sample_data_logs/_search
+{
+  "size": 0,
+  "aggs": {
+    "visits_per_month": {
+      "date_histogram": {
+        "field": "@timestamp",
+        "interval": "month"
+      },
+      "aggs": {
+        "sum_of_bytes": {
+          "sum": {
+            "field": "bytes"
+          }
+        }
+      }
+    },
+    "max_monthly_bytes": {
+      "max_bucket": {
+        "buckets_path": "visits_per_month>sum_of_bytes"
+      }
+    }
+  }
+}
+```
+{% include copy-curl.html %}
+
+## Example response
+
+The aggregation returns the maximum number of bytes from the monthly buckets:
```

Comment:
While it is reasonably intuitive, should explicitly specify what keys indicate. For example - it is array because more than one key might have same value as max

Reply:
Added an extended description in both max and min bucket pages.